src/frontend/components/SøknadsSteg/Kvittering/Kvittering.tsx

```diff
@@ -49,30 +57,39 @@ const Kvittering: React.FC = () => {
         }
     }, []);
 
+    const kvitteringTekster = tekster().KVITTERING;
+
     return (
         <Steg tittel={<TekstBlock block={kvitteringTekster.kvitteringTittel} />}>
-            <KomponentGruppe>
+            <VStack gap="12">
                 <Alert variant="success">
                     <TekstBlock
                         block={kvitteringTekster.soeknadMottatt}
                         flettefelter={{ dato, klokkeslett }}
                     />
                 </Alert>
-            </KomponentGruppe>
-            <KomponentGruppe>
-                <TekstBlock
-                    block={kvitteringTekster.infoTilSoker}
-                    typografi={Typografi.BodyShort}
-                />
-            </KomponentGruppe>
 
-            {varEøsSøknad && (
-                <KomponentGruppe>
-                    <TekstBlock block={kvitteringTekster.kontonummerEOES} />
-                </KomponentGruppe>
-            )}
+                <VStack gap="6">
+                    {alleRelevanteVedleggErSendtInn.current ? (
+                        <TekstBlock
+                            block={kvitteringTekster.trengerIkkeEttersendeVedlegg}
+                            typografi={Typografi.BodyLong}
+                        />
+                    ) : (
+                        <Alert variant="warning">
+                            <TekstBlock block={kvitteringTekster.maaEttersendeVedleggAlert} />
+                        </Alert>
+                    )}
+                    <TekstBlock
+                        // TODO: Endre infoTilSoker i Sanity til å inneholde samme tekst som infoTilSokerMidlertidigTekst.
```

Comment:
Er det meningen å commite inn todoen?

Reply:
Ja, i dette tilfelle må vi endre en Sanity tekst samtidig som koden endres. Den eneste måten å sikre at det gjøres er ved å legge til en midlertidig tekst i Sanity. Deretter må vi endre den originale teksten i Sanity for så å lage en ny PR hvor koden også endres. Jeg legger til en TODO for å sikre at dette ikke blir glemt bort.